_posts/2020-07-28-10-Best-Practices-In-UX-UI-Design.md

```diff
@@ -0,0 +1,185 @@
+* [Clearly show which elements on a page belong together](#3-clearly-show-which-elements-on-a-page-belong-together)
+* [Be honest](#4-be-honest)
+* [Accessibility](#5-accessibility)
+* [Stick to conventions](#6-stick-to-conventions)
+* [Breadcrumbs](#7-breadcrumbs)
+* [Less clicks = better](#8-less-clicks--better)
+* [Dropdown lists](#9-dropdown-lists)
+* [Include visual indicators of length and size](#10-include-visual-indicators-of-length-and-size)
+
+## 1. Company logo placement and usage
+First things first. Your user always needs to find their way back to the homepage in a heartbeat. 
+Providing a 'Home' link in your menu is not enough. 
+It's surprising how often users click on the company logo on the top of your page so make sure it's visible from every page and provide a link behind it to the homepage.
+The company logo is typically placed in the top left corner (on sites designed for left-to-right reading).
+
+<img alt="Logo of Goodsreads is placed in the top left corner and is clickable" src="{{ '/img/2020-07-28-Best-Practices-In-UX-UI-Design/logo-placement.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 1000px;">
+
+## 2. Make clickable elements obvious
+### Buttons
+CTA ([Call To Action](https://en.wikipedia.org/wiki/Call_to_action_(marketing)){:target="_blank" rel="noopener noreferrer"}) buttons need to stand out from the rest of the page so it immediately draws the user's attention.
+Adding a shadow effect to the buttons lets the user know it's clickable and they can interact with it.
+Notice the difference between the following examples:
+
+Example 1:
+<img alt="Clear CTA button" src="{{ '/img/2020-07-28-Best-Practices-In-UX-UI-Design/CTA-colored.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 300px;"> 
+
+Example 2:
+<img alt="Unclear CTA button" src="{{ '/img/2020-07-28-Best-Practices-In-UX-UI-Design/CTA-bland.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 300px;">
+
+In the bottom example the label to save the form is still concise and clear, users should thus still be able to eventually save the data, but it takes a moment longer to register which button to click to make this happen. 
+Remember, the user wants to spend as little time as possible determining the next course of action. 
+
+Let's also discuss the 'Cancel' and 'OK' (or other confirming actions such as 'Save', 'Yes', 'Submit', etc) buttons. 
+So-called secondary actions like 'Cancel' should look the least appealing of the options because it usually isn't the next action the user wants to take.
+Making them the least appealing minimizes the chance of misclicks and nudges the user further towards a successful ending, such as a purchase.
+
+In Windows apps, the 'OK' button should come first and the 'Cancel' button second. 
+On Apple machines however, 'Cancel' will come first and 'OK' second.
```

Comment:
Yes, I think I've read before that the logic of Apple is that it's more logical to have the button on the left represent the cancel/back-action where as in order to proceed to the next action you go to the right.

Reply:
Great remark, I added the throught process behind it to the blog post as well!